### ISSUE TITLE
Breaks rails generate. Rails 7 compat?

### ISSUE THREAD

**Issue by pboling** [slim-template/slim-rails#184]
I am using [`service_actor-rails`](https://github.com/sunny/actor-rails) which provides an `actor` generator, and when I try to use it, while having this `slim-rails` gem installed, it blows up inside `slim-rails`.  It appears to be incompatible with Rails v7?
```
rails generate actor do_something
```

## Error:
```
/Users/pboling/.asdf/installs/ruby/3.1.3/lib/ruby/gems/3.1.0/gems/slim-0.6.1/lib/slim/rails.rb:5:in `<module:TemplateHandlers>': uninitialized constant ActionView::TemplateHandlers::TemplateHandler

    class SlimHandler < TemplateHandler
                        ^^^^^^^^^^^^^^^
Did you mean?  ActionView::TemplateRenderer (NameError)
        from /Users/pboling/.asdf/installs/ruby/3.1.3/lib/ruby/gems/3.1.0/gems/slim-0.6.1/lib/slim/rails.rb:4:in `<module:ActionView>'
        from /Users/pboling/.asdf/installs/ruby/3.1.3/lib/ruby/gems/3.1.0/gems/slim-0.6.1/lib/slim/rails.rb:3:in `<top (required)>'
        from /Users/pboling/.asdf/installs/ruby/3.1.3/lib/ruby/gems/3.1.0/gems/zeitwerk-2.6.6/lib/zeitwerk/kernel.rb:38:in `require'
        from /Users/pboling/.asdf/installs/ruby/3.1.3/lib/ruby/gems/3.1.0/gems/zeitwerk-2.6.6/lib/zeitwerk/kernel.rb:38:in `require'

```

**Comment by pboling** [comment]
OK, I removed the constraints that were added by `bundle add`, and did a generic `bundle update`, and got the following:
```
...
Installing temple 0.8.2 (was 0.9.1)
...
Fetching slim 4.1.0 (was 0.6.1)
Installing slim 4.1.0 (was 0.6.1)
Fetching slim-rails 3.5.1 (was 0.1.2)
Installing slim-rails 3.5.1 (was 0.1.2)
Bundle updated!
```

I think the version of temple in my `Gemfile.lock` that was pre-existing had already eclipsed the older version that slim requires, and so it installed a version of slim so old that it predated the specific version requirement on `temple`.  An oddity of higher specificity. 😆 

Are there plans to loosen the temple restriction, or is it necessary?

**Comment by pboling** [comment]
It is also strange that `!= 0.9.0` disallowed `0.9.1`... Is that expected behavior from bundler?  My impression has been that `!=` was to skip specific versions within an interval of otherwise compatible versions... which would mean it should accept `0.9.1`...

**Comment by pboling** [comment]
Or maybe the latest master hasn't been released.  When I add a discrete `temple` to my Gemfile:
```
gem 'temple', '~> 0.9.1'
```

I get:
```
Bundler could not find compatible versions for gem "temple":
  In Gemfile:
    temple (~> 0.9.1)

    slim (~> 4.1) was resolved to 4.1.0, which depends on
      temple (>= 0.7.6, < 0.9)
```

If that's true then the latest release from 2020-05-08 is unusable for a specific scenario of Bundler + temple.  It looks like the change to support `temple` v0.9.1 was made on October 25, so perhaps a new release is forthcoming?

**Comment by pboling** [comment]
Relevant issues / PRs:
https://github.com/slim-template/slim/pull/894
https://github.com/slim-template/slim/issues/897
